translate/cloud-client/src/test/java/com/example/translate/BatchTranslateTextWithGlossaryAndModelTests.java

```diff
@@ -137,7 +139,7 @@ public void testBatchTranslateTextWithGlossaryAndModel()
         "en",
         "ja",
         INPUT_URI,
-        "gs://" + PROJECT_ID + "/BATCH_TRANSLATION_OUTPUT/",
+        "gs://" + PROJECT_ID + "/" + PREFIX_PATH,
```

Comment:
Ok - but as I look at this, I wonder if String.format should be used as being a bit clearer, I've decided that it's just longer, but I wish we'd get rid of all the string contraction.

Reply:
Updated to use a variable.
```
private static final String OUTPUT_URI =
          String.format("gs://%s/%s/%s/", PROJECT_ID, PREFIX, UUID.randomUUID());
```